api/src/main/java/org/apache/iceberg/UpdateSchema.java

```diff
@@ -361,4 +361,27 @@ default UpdateSchema updateColumn(String name, Type.PrimitiveType newType, Strin
    *                                  change conflicts with other changes.
    */
   UpdateSchema moveAfter(String name, String afterName);
+
+
+  /**
+   * Applies all field additions and updates from the provided new schema to the existing schema so
```

Comment:
@rdblue is this javadoc overdoing it a bit?

Reply:
Looks good to me.